assignment-day-2/src/pipeline/extract_timesheet_data.py

```diff
@@ -0,0 +1,39 @@
+from utils import connect
+
+try:
+    connection = connect()
+    cursor = connection.cursor()
+
+    def truncate_timesheet_data():
```

Comment:
Maybe we can use the same function to truncate both employee and timesheet data by using as a constant as I have mentioned in the suggestion of truncate_employee_data.sql.